config/locales/en.yml

```diff
@@ -0,0 +1,8 @@
+# English strings go here for Rails i18n
+en:
+  label_last_seen_issues_general: 'General'
+  number_of_issues_displayed: 'Number of issues displayed'
+  number_of_issues_displayed_explanation: 'Represents the number of how many issues to be displayed in pop-up. E.g: 5'
+  label_last_seen_issues_menu_entry: "Last seen issues"
+  history_missing_message: "There is no history for last seen issues!"
+  history_error_message: "An internal server error has occurred! Please check browser console for more details!"
```

Comment:
Ok for the message (I see you removed the shortcut for the browser console, which is good).

I'm not sure if we should add the `Please check browser console for more details` but I don't think we need to change it now.

Reply:
I've removed `Please check browser console for more details`.